src/systemverilog/systemverilog.ts

```diff
@@ -393,21 +391,16 @@ export const language = <ILanguage>{
 		],
 
 		string: [
-			[/[^\\"]+/, 'string'],
-			[/@escapes/, 'string.escape'],
-			[/\\./, 'string.escape.invalid'],
-			[/"/, 'string', '@pop']
+			[/"/, 'string.escape', '@stringBody']
 		],
-
-		raw: [
-			[/(.*)(\))(?:([^ ()\\\t]*))(\")/, {
-					cases: {
-						'$3==$S2': ['string.raw', 'string.raw.end', 'string.raw.end', { token: 'string.raw.end', next: '@pop' }],
-						'@default': ['string.raw', 'string.raw', 'string.raw', 'string.raw']
-					}
+		stringBody: [
+			[/\\./, {
```

Comment:
I don't think we need the case statement here, right?  Does this work?
`[/\\./, 'error-token']`